genon/preprocessor/facade/parser_processor.py

```
@@ -0,0 +1,1520 @@
+            overlap_start_ms = start_ms - 300 if start_ms > 0 else start_ms
+            end_ms = start_ms + chunk_len
+            audio_chunk = audio[overlap_start_ms:end_ms]
+            audio_chunk.export(os.path.join(self.tmp_path, "tmp_{}.wav".format(str(i))), format="wav")
+        return glob(os.path.join(self.tmp_path, "*.wav"))
+
+    def transcribe_audio(self, file_path_lst: list):
+        transcribed_text_chunks = []
+
+        def _send_request(filepath: str):
+            files = {'file': (filepath, open(filepath, 'rb'), 'audio/mp3')}
+            response = requests.post(self.req_url, data=self.req_data, files=files)
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)